ManageWikiExtensions.php

```diff
@@ -1182,6 +1182,19 @@
 			'requires' => [],
 			'section' => 'specialpages',
 		],
+		'createdpageslist' => [
+			'name' => 'CreatedPagesList',
+			'linkPage' => 'https://www.mediawiki.org/wiki/Extension:CreatedPagesList',
+			'var' => 'wmgUseCreatedPagesList',
+			'conflicts' => false,
+			'requires' => [],
+			'section' => 'specialpages',
+			'install' => [
+				'sql' => [
+					'createdpageslist' => '$IP/extensions/CreatedPagesList/sql/patch-createdpageslist.sql'
```

Comment:
You'll need to use "" instead of '' for the SQL path.

Reply:
Done.